internal/mysql/node.go

```diff
@@ -431,7 +431,8 @@ func (n *Node) GetCrashRecoveryTime() (time.Time, error) {
 	s := bufio.NewScanner(fh)
 	for s.Scan() {
 		line := s.Text()
-		if strings.HasSuffix(line, " Starting crash recovery...") {
+		if strings.HasSuffix(line, "[Note] Starting crash recovery...") || // For 5.7 version
```

Comment:
May be we should check strings.Contains(line, "Starting crash recovery.") here?

Reply:
If some line will contain string "Starting crash recovery." for some reason - it will start crash recovery in mysync. This implementation reduces the chance of accidental triggering